src/data/charts.json

```diff
@@ -855,6 +957,9 @@
       {
         "title": "Cereal yield in kg per hectare (Thousands)",
         "visual": {
+          "props": {
+            "horizontal": true
```

Comment:
What happened to setting `horizontal` depending on data length?

Reply:
@kilemensi is still a thing, for now this has to be manually defined. 

Using ChartFactory from Dominion.